bin/cli.ts

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env node
+
+import cac from 'cac';
+import { script, Completion } from '../src/index.js';
+import tab from '../src/cac.js';
+
+import { setupCompletionForPackageManager } from './completion-handlers';
+
+const packageManagers = ['npm', 'pnpm', 'yarn', 'bun'];
+const shells = ['zsh', 'bash', 'fish', 'powershell'];
+
+const cli = cac('tab');
+
+cli
+  .command(
+    '<packageManager> complete',
+    'Process completion requests from shell'
+  )
+  .allowUnknownOptions()
```

Comment:
why do we allow unknown options? since all the unknown options would come after the --, so that'd be allowed even without this flag!